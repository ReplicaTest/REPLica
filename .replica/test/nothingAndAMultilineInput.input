"ab" ++  "cd"
"ab" <+> "cd"
:q
