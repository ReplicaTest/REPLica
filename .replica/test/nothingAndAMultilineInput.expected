Main> "abcd"
Main> "abcd"
Main> Bye for now!
